inc/namespace.php

```diff
@@ -196,3 +198,77 @@ function enqueue_scripts() {
 	// Print queued scripts.
 	print_head_scripts();
 }
+
+/**
+ * Schedule index maintenance daily.
+ */
+if( ! wp_next_scheduled( __NAMESPACE__ . '\\index_maintenance' ) ) {
+	wp_schedule_event( time(), 'daily',  __NAMESPACE__ . '\\index_maintenance' );
+}
```

Comment:
Again - you should avoid calling functions directly in a file, what I mean by mixing side effects and declaring functions is something like the following:

```php
// declaring a function
function foo() {
    return 'foo';
}
// calling a function
bar();
```

In this case you should move this block of code inside the `setup()` function.